pom.xml

```diff
@@ -678,6 +681,12 @@
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
         </dependency>
+        
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-shaded-client</artifactId>
+            <version>${hbase.client.version}</version>
```

Comment:
Please use the provide scope, otherwise, ShardingSphere's dependencies will become heavier and heavier.